PyISY/Connection.py

```diff
@@ -154,10 +154,22 @@ def getNodes(self):
         result = self.request(req_url)
         return result
 
+    # Get the device notes, currently only the Spoken property is saved.
     def getNodeNotes(self,nid):
         req_url = self.compileURL(['nodes', nid, 'notes'])
         result = self.request(req_url)
-        return result
```

Comment:
That makes sense, but I would like a common method to parse the xml and return a dict so it can be called by node and group to set their _notes so it's not inside both classes?  Currently it only pulls out the spoken param, but I may soon want to grab more of the info, and UDI may add more info to notes and having a common routine would seem to make sense?